app/views/contact_mailer/contact_confirmation.text.erb

```diff
@@ -0,0 +1,11 @@
+Hi <%= @contact.name %>,
+
+You are receiving this email because you have contacted us using the contact form at Boilerplate's website: https://https://boilerplate-app-cms.herokuapp.com/
```

Comment:
The website link has an extra https in it, we might want to have this be dynamic like we have in the other password reset email ticket